packages/react-storage/src/components/StorageBrowser/controls/hooks/useStatusDisplay.ts

```diff
@@ -1,20 +1,25 @@
 import { StatusDisplayProps } from '../../composables/StatusDisplay';
 import { useControlsContext } from '../../controls/context';
-import { displayText } from '../../displayText/en';
 
 export const useStatusDisplay = (): StatusDisplayProps => {
   const { data } = useControlsContext();
-  const { statusCounts } = data;
+  const {
+    statusCounts,
+    statusDisplayCanceledLabel,
+    statusDisplayCompletedLabel,
+    statusDisplayFailedLabel,
+    statusDisplayQueuedLabel,
+  } = data;
 
   if (!statusCounts?.TOTAL) {
     return { statuses: [], total: 0 };
   }
 
   const statuses = [
-    { name: displayText.statusDisplayCompleted, count: statusCounts.COMPLETE },
-    { name: displayText.statusDisplayFailed, count: statusCounts.FAILED },
-    { name: displayText.statusDisplayCanceled, count: statusCounts.CANCELED },
-    { name: displayText.statusDisplayQueued, count: statusCounts.QUEUED },
+    { name: statusDisplayCompletedLabel!, count: statusCounts.COMPLETE },
+    { name: statusDisplayFailedLabel!, count: statusCounts.FAILED },
+    { name: statusDisplayCanceledLabel!, count: statusCounts.CANCELED },
+    { name: statusDisplayQueuedLabel!, count: statusCounts.QUEUED },
```

Comment:
I think it's probably safer to fallback to empty string? If this control were used somewhere where the context did not provide these, we wouldn't want it to just blow up